diff --git a/roles/etcd/templates/etcd.j2 b/roles/etcd/templates/etcd.j2
index b078057..4385ad6 100644
--- a/roles/etcd/templates/etcd.j2
+++ b/roles/etcd/templates/etcd.j2
@@ -3,6 +3,7 @@
   --restart=on-failure:5 \
   --env-file=/etc/etcd.env \
   --net=host \
+  -e ETCD_UNSUPPORTED_ARCH=arm64 \
   -v /etc/ssl/certs:/etc/ssl/certs:ro \
   -v {{ etcd_cert_dir }}:{{ etcd_cert_dir }}:ro \
   -v {{ etcd_data_dir }}:{{ etcd_data_dir }}:rw \
diff --git a/roles/kubernetes/node/templates/kubelet.j2 b/roles/kubernetes/node/templates/kubelet.j2
index 14d7e0d..667e558 100644
--- a/roles/kubernetes/node/templates/kubelet.j2
+++ b/roles/kubernetes/node/templates/kubelet.j2
@@ -20,6 +20,7 @@ KUBELET_HOSTNAME="--hostname-override={{ kube_override_hostname }}"
 --enable-cri={{ kubelet_enable_cri }} \
 {% endif %}
 --cgroups-per-qos={{ kubelet_cgroups_per_qos }} \
+--cgroup-driver=systemd \
 --enforce-node-allocatable={{ kubelet_enforce_node_allocatable }} {% endif %}{% endset %}
 
 {# DNS settings for kubelet #}
